src/foam/nanos/auth/User.js

```diff
@@ -175,7 +175,7 @@ foam.CLASS({
       width: 100,
       tableWidth: 160,
       validateObj: function(organization) {
-        if (!(organization.trim())) {
+        if ( ! (organization.trim()) ) {
```

Comment:
Side question, why do we say 'Company' name is required for 'Organization'? organization is perfectly correct term on it's own.  For an an employee 'user' the organization is the entity that employs them.  We have businessName for business/company.

Reply:
lol. Nice attention to detail. What you are saying makes sense to me. So looks like companyName is associated to the mapping to businessName. And somewhere along the way organization was used as businessName as well. So now they are the same :(

Yikes this should be cleaned up. If you give the thumbs up @jlhughes I'll make a ticket that will hopefully clean this all up.

I will change this right now.

Reply:
re: Organization / BussinessName - yes, please create a ticket.  Suspect this will impact many views. as some onboarding views only test for organization and not businessname.